boards/px4/fmu-v5/default.cmake

```
@@ -18,6 +18,7 @@ px4_add_board(
 	DRIVERS
 		adc
 		barometer # all available barometer drivers
+        rotor_frequency
```

```suggestion
		rotor_frequency
```

Can you keep the list sorted alphabetically?

---

Sure, repaired in new commit